label_options/unc-ebike-label-options.json

```diff
@@ -0,0 +1,100 @@
+        {"value": "bike", "baseMode":"BICYCLING", "met_equivalent":"BICYCLING", "kgCo2PerKm": 0},
+        {"value": "bus", "baseMode": "BUS", "met_equivalent": "IN_VEHICLE", "kgCo2PerKm": 0.20727},
+        {"value": "taxi", "baseMode":"TAXI", "met_equivalent":"IN_VEHICLE", "kgCo2PerKm": 0.30741},
+        {"value": "ridehail", "baseMode":"TAXI", "met_equivalent":"IN_VEHICLE", "kgCo2PerKm": 0.30741},
+        {"value": "train", "baseMode": "TRAIN", "met_equivalent":"IN_VEHICLE", "kgCo2PerKm": 0.12256},
+        {"value": "air", "baseMode":"AIR", "met_equivalent":"IN_VEHICLE", "kgCo2PerKm": 0.09975},
+        {"value": "not_a_trip", "baseMode":"UNKNOWN", "met_equivalent":"UNKNOWN", "kgCo2PerKm": 0},
+        {"value": "other", "baseMode":"OTHER", "met_equivalent":"UNKNOWN", "kgCo2PerKm": 0}
+    ],
+    "PURPOSE": [
+      {"value": "home"},
+      {"value": "work"},
+      {"value": "school"},
+      {"value": "shopping"},
+      {"value": "medical"},
+      {"value": "friends_family"},
+      {"value": "religious"},
+      {"value": "restaurant"},
+      {"value": "entertainment"},
+      {"value": "recreation"},
+      {"value": "pudo"},
+      {"value": "accbus"},
+      {"value": "acctrainair"},
+      {"value": "other"}
+    ],
+    "REPLACED_MODE": [
+        {"value": "no_travel"},
+        {"value": "car"},
+        {"value": "walk"},
+        {"value": "e-bike"},
+        {"value": "bike"},
+        {"value": "bus"},
+        {"value": "taxi"},
+        {"value": "ridehail"},
+        {"value": "train"},
+        {"value": "air"},
+        {"value": "other"}
+    ],
+    "translations": {
+      "en": {
+        "car": "Car",
+        "walk": "Walk",
+        "bike": "Regular bike",
+        "ebike": "E-bike",
+        "bus": "Bus",
+        "taxi": "Taxi",
+        "ridehail": "Uber / Lyft",
+        "train": "Train / Amtrak",
+        "air": "Airplane",
+        "not_a_trip": "Not a trip",
+        "other": "Other",
+        "home": "Home",
+        "work": "Work",
+        "school": "School",
+        "shopping": "Shopping",
+        "medical": "Medical",
+        "friends_family": "Visiting friends or family",
+        "religious": "Church or house of worship",
+        "restaurant": "Restaurant",
+        "entertainment": "Entertainment, sightseeing, etc.",
+        "recreation": "Recreation / exercise",
+        "pudo": "Pick up / drop off a person",
+        "accbus": "Get to / from bus stop",
+        "acctrainair": "Get to / from train station or airport",
+        "no_travel": "Would not have made trip"
+      },
+      "es": {
+        "car": "Car",
+        "walk": "Walk",
+        "bike": "Regular bike",
+        "ebike": "E-bike",
+        "bus": "Bus",
+        "taxi": "Taxi",
+        "ridehail": "Uber / Lyft",
+        "train": "Train / Amtrak",
+        "air": "Airplane",
+        "not_a_trip": "Not a trip",
+        "other": "Other",
+        "home": "Home",
+        "work": "Work",
```

Comment:
Shouldn't this be in Spanish?

Reply:
Yes, we are in the process of hiring a translator so this is just a placeholder for now